### DISCUSSION TITLE
"Login as" feature

### DISCUSSION THREAD

**Discussion by en0tuk** [pelican-dev/panel#266] [Ideas]
Suggestion: Add ability to login as another user from admin panel.

**Comment by Poseidon281** [comment]
Why would you want to do that?

**Reply by Sharktheone** [reply]
Probably to verify that everything was set up correctly and the user has all privileges they need (and no additional ones)

Discord has this feature with their role system, since Pelican wants to have a more complex permission system, I think it could be a great addition.
